### ISSUE TITLE
Overfitting problem with model

### ISSUE THREAD

**Issue by skrealworld** [RITficialIntelligence/kaggle-cancer#12]
Right now model is overfitting by huge margin, we need to investigate why and look for the solutions. 
Right direction for solutions would be: 
-Instead of re-training only last layer, retrain more layers. 
-Add dropout ? 
-Batch normalization ? 
- Some feature engineering ? 
- More data ? 
- Data augmentation ?
